src/vivarium_gates_mncnh/data/hemoglobin_effects/hgb_birth_effect_generation.py

```diff
@@ -124,12 +124,18 @@ def load_prepped_rrs(outcome):
     """Load relative risks of hemoglobin on specific outcome that are:
     - rescaled to a tmrel of 120 g/L,
     - interpolated to the exposure levels used in GBD,
-    - reordered by magnitude of risk at the lowest exposure level."""
+    - reordered by magnitude of risk at the lowest exposure level,
+    - and scale up to 500 draws by duplicating existing draws."""
     rrs = load_bop_rrs(outcome)
     exposure_levels = get_gbd_exposure_levels()
     rrs = convert_rrs_to_gbd_exposure(rrs, exposure_levels)
     rrs["outcome"] = outcome
     rrs = transform_and_reorder_rrs(rrs, exposure_levels)
+    rrs_copy = rrs.copy().set_index([x for x in rrs.columns if "draw" not in x])
```

Comment:
Is this draw-copying documented somewhere?

Reply:
New PR here! https://github.com/ihmeuw/vivarium_research/pull/1795